spec/controllers/v1.0/actions_controller_spec.rb

```diff
@@ -1,2 +1,38 @@
-RSpec.describe Api::V1x0::ActionsController, :type => :controller do
+describe Api::V1x0::ActionsController, :type => :request do
+  let(:encoded_user) { encoded_user_hash }
+  let(:request_header) { { 'x-rh-identity' => encoded_user } }
+  let(:tenant) { create(:tenant, :external_tenant => 369_233) }
+
+  let!(:template) { create(:template) }
+  let(:template_id) { template.id }
+  let!(:workflows) { create_list(:workflow, 16, :template_id => template.id) }
+
+  let(:api_version) { version }
+
+  context "with entitlement enforcement" do
+    let(:false_hash) do
+      false_hash = default_user_hash
+      false_hash["entitlements"]["hybrid_cloud"]["is_entitled"] = false
+      false_hash
+    end
+    let(:missing_hash) do
+      missing_hash = default_user_hash
+      missing_hash.delete("entitlements")
+      missing_hash
+    end
+
+    it "fails if the hybrid_cloud entitlement is false" do
+      headers = { 'x-rh-identity' => encoded_user_hash(false_hash), 'x-rh-insights-request-id' => 'gobbledygook' }
+      get "#{api_version}/workflows", :params => { :limit => 5, :offset => 0 }, :headers => headers
```

Comment:
Does not make sense that you test a `workflow` endpoint in the `actions_controller_spec`.
You can move the tests to any of the existing spec under `requests` folder, where you don't need to prepare variables in order to get an endpoint. 
Later we should move all specs from `requests` to `controllers`.

cc @hsong-rh

Reply:
@bzwei I'm not testing workflow specific functionality (any endpoint would work, I just picked workflows), it's testing `actions_controller.rb` functionality, specifically stuff in the `IndexMixin`. If you want me to move it to the workflows spec file I can, but just thought I would explain why I put this here instead of there.

Reply:
@lindgrenj6 Please put this temporarily into one of spec file in `spec/requests`. Later I'll move all of them back to `controllers`.